src/locale/locales/fr/messages.po

```diff
@@ -6528,24 +6312,16 @@ msgstr "Champ de saisie de texte"
 
 #: src/components/dialogs/VerifyEmailDialog.tsx:82
 msgid "Thank you! Your email has been successfully verified."
-msgstr ""
+msgstr "Merci ! Votre e-mail a été vérifié avec succès."
 
 #: src/components/dms/ReportDialog.tsx:129
 #: src/components/ReportDialog/SubmitView.tsx:82
 msgid "Thank you. Your report has been sent."
 msgstr "Nous vous remercions. Votre rapport a été envoyé."
 
-#: src/components/dialogs/nuxs/TenMillion/index.tsx:593
-#~ msgid "Thanks for being one of our first 10 million users."
-#~ msgstr "Merci d’avoir été l’une des 10 premières millions de personnes à s’inscrire à Bluesky."
-
-#: src/components/intents/VerifyEmailIntentDialog.tsx:74
-#~ msgid "Thanks, you have successfully verified your email address."
-#~ msgstr "Merci, vous avez vérifié avec succès votre adresse e-mail."
-
 #: src/components/intents/VerifyEmailIntentDialog.tsx:82
 msgid "Thanks, you have successfully verified your email address. You can close this dialog."
-msgstr ""
+msgstr "Merci, vous avez vérifié avec succès votre adresse e-mail. Vous pouvez fermer cette fenêtre."
```

Comment:
`dialog` has been translated elsewhere as `boîte de dialogue`, is `fenêtre` a better choice here for brevity, or perhaps due to the different style of the dialog (a bottom sheet here IIRC)?

Reply:
Mainly for brievety, and the fact that `boîte de dialogue` is quite a verbose term and not that common term in UI copy in 2024.

It wasn't a problem for an accessibility label to use it — after all, screenreader users are a little bit more used to those terms, as we need more explicit terms to qualify UI elements in text — but in for a common user, `fenêtre` is way more common than `boîte de dialogue`. (It may also be the case in English tho!)

Reply:
Makes sense to me 👌